pkg/sql/plan/function/function_id.go

```diff
@@ -387,6 +387,10 @@ const (
 	// FUNCTION_END_NUMBER is not a function, just a flag to record the max number of function.
 	// TODO: every one should put the new function id in front of this one if you want to make a new function.
 	FUNCTION_END_NUMBER
+	EMBEDDING
```

Comment:
First you should NEVER add anything behind FUNCTION_END_NUMBER.     Read the comment.

Next, please add a prefix to your "internal helper" functions.   Call it LLM_EMBEDDING, LLM_CHUNK, and maybe LLM_EXTRACT_TEXT.   The extract_text can be general.

Reply:
Have relocated and renamed the helper functions.